data/persistence/build.gradle.kts

```diff
@@ -30,61 +30,67 @@ android {
     }
 }
 
+fun isIdea(): Boolean = System.getProperty("idea.fatal.error.notification") != null
```

Comment:
Do we need this function in every module? There should be a nicer way

Reply:
Maybe can solve this from the project level gradle

Reply:
On it already...but I bumped into some other issues....sooooo

Reply:
Now you take a look again